src/main/scala/viper/gobra/frontend/info/implementation/typing/ghost/GhostExprTyping.scala

```diff
@@ -280,6 +281,7 @@ trait GhostExprTyping extends BaseTyping { this: TypeInfoImpl =>
       case n: PInvoke => (exprOrType(n.base), resolve(n)) match {
         case (Right(_), Some(_: ap.Conversion)) => false // Might change at some point
         case (Left(callee), Some(p: ap.FunctionCall)) => go(callee) && p.args.forall(go)
+        case (Left(_), Some(p: ap.PredExprInstance)) => !strong
         case _ => false
```

Comment:
It seems to me that the case for predicate instances is also missing:
`case (Left(_), Some(p: ap.PredicateCall)) => !strong`